evaluation/pipeline_invoker.py

```
@@ -0,0 +1,126 @@
+    ) -> None:
+        self._plugin_type = plugin_type
+        self._config = config
+        self._body_of_knowledge_id = body_of_knowledge_id
+        self._plugin = None
+        self._tracing_store: TracingKnowledgeStore | None = None
+        self._llm_adapter = None
+
+    async def setup(self) -> None:
+        """Initialize the pipeline: container, adapters, plugin."""
+        from core.adapters.chromadb import ChromaDBAdapter
+        from core.adapters.openai_compatible_embeddings import OpenAICompatibleEmbeddingsAdapter
```

Flake8 (7.3.0)

[error] 'core.adapters.openai_compatible_embeddings.OpenAICompatibleEmbeddingsAdapter' imported but unused

(F401)